includes/wikia/services/tests/ImagesServiceUploadTest.php

```diff
@@ -31,11 +31,6 @@ protected function setUp() {
 		$this->fileHash = md5( Http::get( self::URL, 'default', ['noProxy' => true] ) );
 		$this->reuploadedFileHash = md5( Http::get( self::REUPLOAD_URL, 'default', ['noProxy' => true] ) );
 
-		// use Swift domain
-		global $wgDevelEnvironmentName;
-		// Disabling forcing the image domain due to BAC-1136
-		//$this->mockGlobalVariable( 'wgDevBoxImageServerOverride', "static.{$wgDevelEnvironmentName}.wikia-dev.com" );
-
 		// debug
 		global $wgLocalFileRepo;
 		echo "Files repository in use: '{$wgLocalFileRepo['backend']}'\n"; // local-backend / swift-backend
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)